docs/release-notes/v11.1.0.md

```diff
@@ -0,0 +1,139 @@
+---
+id: v11.1.0
+title: Version v11.1.0 Release Notes
+sidebar_label: v11.1.0
+image: "img/release-note-link-preview.png"
+---
+
+_May 25th, 2023_
+
+## Console
+
+### New Features
+
+#### User Groups Management
+
+From a dedicated section inside of the Company Overview, Company Owners can now manage Company users also through Groups.  
+Thanks to Groups it is possible to simplify the management of a multitude of users who have to be assigned the same privileges (e.g. a work team).  
+Management of users through groups sits alongside the actual individual management of users, to allow you to better structure access to resources as you prefer, giving you the freedom to choose how to govern access levels of your Company users.  
+
+To discover more about user groups management, just take a look at the related [documentation page](../development_suite/identity-and-access-management/group-management.md). 
+
+# Privatized Marketplace for Companies
+
+From now on, Companies can have their own private Marketplace items, which will not be visible and usable from those who are not part of those specific Companies.   
+More specifically, Company Owners and Project Administrators can now be able to manage – directly from CMS – Marketplace resources that will be available exclusively for the members of their Company. This has to be done 
+by specifying the `tenantId` as a Company identifier. 
+
+To discover more information, visit the dedicated Marketplace [documentation page](../marketplace/add_to_marketplace/contributing_overview#public-and-private-marketplace.md).
+
+### Improvements
+
+### Bug Fix
```

Comment:
@marcofilippi @albertotessarotto are we doing again the paragraph for each bug fixes? Why don't we write the list as in the past release notes?

I need to add a bugfix for the backoffice configurator, I need to understand where to put it

Reply:
agree, we now modify the structure of the Bugfix section to make it the same as in the previous rn

Reply:
I modified, give a check if it is okay for you @TheRusCompass